apiserver/internal/handlers/resources/resourcemigration.go

```diff
@@ -138,8 +138,6 @@ func (h *resourcesMigrationUploadHandler) processPost(
 		RetrievedByType: retrievedByType,
 		Size:            details.size,
 		Fingerprint:     details.fingerprint,
-		Origin:          details.origin,
```

Comment:
Sorry to be a stick in the mud, but I think we should still check that these are correct. Here we are getting the resource UUID from the resource name and application name, then attaching the blob to that resource. We're not actually making sure that it is indeed the resource we think it is. i.e. the one that was set on the application when the database migration was done.

We would be relying on the migration source controller to upload the right blob. Which is should, but since it is telling us which origin and revision it is sending over, I think we should check it.